school.rb

```diff
@@ -0,0 +1,51 @@
+class School
```

Comment:
You have `School` as a superclass of `Student` and `Instructor`.  Can you say that a Student is a School, or that an Instructor is a School?  What's a noun that actually encompasses both Students and Teachers?